Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -60,107 +62,121 @@ async function run() {
             resourceGroupName = await azureRESTUtility.getResourceGroupName(endPoint, webAppName);
         }
 
-        var publishingProfile = await azureRESTUtility.getAzureRMWebAppPublishProfile(endPoint, webAppName, resourceGroupName, deployToSlotFlag, slotName);
-        console.log(tl.loc('GotconnectiondetailsforazureRMWebApp0', webAppName));
+        // For container based linux deployment
+        if(webAppKind && webAppKind === "linux")
```

Comment:
This condition will break the ability to do in-built image deployment to Linux web apps. Can you check condition that is specific to container based deployment?